src/pages/nft/class/_classId/_nftId.vue

```diff
@@ -89,6 +74,72 @@
+          <div v-if="!nftIsNew" class="hidden col-span-2 flex-col items-center gap-[24px] phone:flex">
+            <div
+              v-if="messageList.length && messageList[0].message"
+              class="flex flex-col items-center justify-center w-full py-[24px] border-[2px] border-shade-gray rounded-[24px]"
+            >
+              <Label :text="$t('nft_message_type_generic')" class="text-medium-gray" />
+              <ul v-if="creatorMessage.message" class="flex flex-col gap-[24px] w-full px-[24px]">
+                <NFTMessage
+                  :key="`${creatorMessage.txHash}-${creatorMessage.event}`"
+                  :type="creatorMessage.event"
+                  :from-type="creatorMessage.fromType"
+                  :from-wallet="creatorMessage.fromWallet"
+                  :to-type="creatorMessage.toType"
+                  :to-wallet="creatorMessage.toWallet"
+                  :message="creatorMessage.message"
+                  :message-type="creatorMessage.messageType"
+                  :is-list="false"
+                  tag="li"
+                />
+              </ul>
+              <ButtonV2
+                v-if="shouldShowMessageButton"
+                preset="outline"
+                class="mt-[16px] mb-[8px]"
+                :text="$t('nft_details_page_button_view_message', {
+                  num: messageList.length
+                })"
+                :to="{
+                  name: 'nft-class-classId-nftId-message',
+                  params: { classId, nftId: selectedNFTId }
+                }"
+                @click.native="handleClickViewMessage"
+              >
+                <template #prepend>
+                  <IconView />
+                </template>
+                <template #append>
+                  <IconArrowLeft class="rotate-[180deg]" />
+                </template>
+              </ButtonV2>
+            </div>
+            <NFTPageNFTSelect
+              :nft-collector-collected-count="nftCollectorCollectedCount"
+              :selected-nft-id="selectedNFTId"
+              :nft-collector-collected-nft-list="nftCollectorCollectedNFTList"
+              @select="onSelectNFT"
+            />
+            <div v-if="messageList.length === 1 && !creatorMessage.message">
+              <ul class="flex flex-col gap-[24px] w-full laptop:px-[24px]">
+                <NFTMessage
+                  :key="`${creatorMessage.txHash}-${creatorMessage.event}`"
+                  :type="creatorMessage.event"
+                  :from-type="creatorMessage.fromType"
+                  :from-wallet="creatorMessage.fromWallet"
+                  :to-type="creatorMessage.toType"
+                  :to-wallet="creatorMessage.toWallet"
+                  :message="creatorMessage.message"
+                  :message-type="creatorMessage.messageType"
+                  :is-list="true"
+                  tag="li"
+                />
+              </ul>
+            </div>
```

Comment:
Should not display creator message if `!creatorMessage.message` and v-if should put on `<ul/>` instead of extra `<div/>`?